Makefile

```diff
@@ -53,7 +57,6 @@ test-no-testing-data: in
 	@MNE_SKIP_TESTING_DATASET_TESTS=true \
 	$(NOSETESTS) mne
 
-
 test-no-sample-with-coverage: in testing_data
 	rm -rf coverage .coverage
 	@MNE_SKIP_SAMPLE_DATASET_TESTS=true \
```

Comment:
and it can actually be removed from the Makefile as well, we have no sample dataset tests